packages/core/src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -56,6 +57,7 @@ export async function buildConnection(
     provider,
     types: CUSTOM_TYPES,
   })
+  await cryptoWaitReady()
```

Comment:
Why do we need it in `buildConnection`?

Reply:
we don't, but I though it's a nice feature. Shall I remove?

Reply:
or move it to Kilt.connect() maybe?

Reply:
If you want to add this convenience, I think `Kilt.connect()` could be a good place, because when you have an application, which disconnects the chain, whenever it goes into background mode, you will call Kilt.connect to reconnect. On the other hand, an application might just want to leverage the implicit connections...

Since it is only connected to the Identities, adding it to blockchain connection related functions feels like the wrong place, though. I guess I would remove it here, but add a disclaimer in the documentation, that you have to call it, whenever you want to do something with the identities.

Can we not just call it, whenever we do identity functions?

Reply:
removed it, as discussed